### ISSUE TITLE
Impossible to open a link in a new tab on the "clickable link block" if the item is not selected

### ISSUE THREAD

**Issue by Massedil** [FreshRSS/FreshRSS#3120]
Context :

I sometimes want to open a link in a new tab (right click > open in a new tab) or to copy the article URL without opening the article on FreshRSS.

Actual :

- The link is not in the "full clickable" zone before opening the article.
- The link is on the "full clickable" zone after opening the article (article title is bold).

This is not comfortable if the title of the article is short.

Expected :

- The link is always in the full clickable zone (without the need to open the article in FresshRSS).

A screenshot with the zone where the link is clickable (right click > open in a new tab) on Firefox Nightly : 

![](https://user-images.githubusercontent.com/20357783/87683652-a1f33280-c781-11ea-9822-a6952cf5b173.png)

**Comment by Frenzie** [comment]
While https://github.com/FreshRSS/FreshRSS/issues/2344 is technically about something slightly different, the way https://github.com/FreshRSS/Extensions/tree/master/xExtension-TitleWrap is implemented takes care of that and whatever @marienfressinaud will come up with likely will too.

**Comment by math-GH** [comment]
It is strange. The `<a>` is wide enough, but if the article is unread then only the title has the link, the empty space after the title not (but it is still the `<a>`.

**Comment by math-GH** [comment]
I would say that this issue was already solved in the past.
@Massedil Could you please double check if it is fine now? Thanks!

**Comment by Massedil** [comment]
> I would say that this issue was already solved in the past.
> @Massedil Could you please double check if it is fine now? Thanks!

Still here for me in 1.21.0.

EDIT : still here in 1.23.1.

**Comment by math-GH** [comment]
Which theme do you use? Are extensions installed?

**Comment by Massedil** [comment]
> Which theme do you use? Are extensions installed?

Origine — Par : Marien Fressinaud
Le thème par défaut pour FreshRSS
8/11

No extension except "Tumblr-GDPR" enabled by default.

**Comment by Massedil** [comment]
Because of this for the `li` around the `a` :

```
.flux:not(.current):hover .item.title {
  background-color: inherit;
  max-width: calc(100% - 320px);
  position: absolute; /* When hovering it becomes "position: absolute;" and only the text becomes clickable */
}
```

![2024-03-21_15-29](https://github.com/FreshRSS/FreshRSS/assets/20357783/dada7d11-f7cd-4111-b851-7e2513ede3a9)
![2024-03-21_15-31](https://github.com/FreshRSS/FreshRSS/assets/20357783/051e80e3-2051-402d-9114-36292f07f222)

**Comment by math-GH** [comment]
Great. Thanks. I can reproduce it now.

(Side note: I the "TitleWrap" extension is installed and enabled then it is not an issue. Maybe it could help to have a quick workaround)

**Comment by Alkarex** [comment]
Hum, I cannot reproduce it on my side, with Firefox, Origin theme, no extension

**Comment by math-GH** [comment]
@Massedil Please test this fix, if possible: #6444

**Comment by Massedil** [comment]
I tried to use `git apply 6444.patch` (from [here](https://patch-diff.githubusercontent.com/raw/FreshRSS/FreshRSS/pull/6444.patch)) to quickly update my local code, but I'm on the release channel (1.23.1) and it fails to apply or I miss something. Do you know how I can test and revert it quickly knowing I installed FreshRSS without git ?

**Comment by math-GH** [comment]
> I tried to use `git apply 6444.patch` (from [here](https://patch-diff.githubusercontent.com/raw/FreshRSS/FreshRSS/pull/6444.patch)) to quickly update my local code, but I'm on the release channel (1.23.1) and it fails to apply or I miss something. Do you know how I can test and revert it quickly knowing I installed FreshRSS without git ?

it is now merged into `edge` (`1.23.2-dev`)

**Comment by Massedil** [comment]
I can confirm this issue is solved in 1.24.0, thanks !
